tests/inference_runner.py

```diff
@@ -1,32 +1,49 @@
 from frontend.pre_analysis import PreAnalyzer
+from frontend.stmt_inferrer import *
+from frontend.stubs.stubs_handler import StubsHandler
 import frontend.z3_types as z3_types
 import ast
+import sys
 
-r = open("tests/inference/expressions_test.py")
+r = open("tests/inference/classes_test.py")
 t = ast.parse(r.read())
 
 analyzer = PreAnalyzer(t)
 
-z3_types.init_types({
-    "max_tuple_length": analyzer.maximum_tuple_length(),
-    "max_function_args": analyzer.maximum_function_args()
-})
-
-from frontend.stmt_inferrer import *
 
+config = analyzer.get_all_configurations()
+solver = z3_types.TypesSolver(config)
+stubs_handler = StubsHandler()
```

Comment:
It seems that this (or something else somewhere) adds all the axioms for all the built-in types to the z3 solver. This might be the reason why everything is slow. Can we just add the axioms that correspond to the built-in types that we need for the program that we are analyzing?